.github/workflows/build-publish.yml

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+on:
+    push:
+      paths-ignore:
+      - '**.md'
+  
+jobs:
+  build:
+    env:
+      IMAGE_NAME: quay.io/che-incubator/che-api-sidecar
```

Comment:
Shouldn't it be che-incubator/che-editor-intellij-community ?

Reply:
yes sure, was in the middle of copy/paste :-)